doc/stages/readers.ept.rst

```diff
@@ -39,6 +39,9 @@ filename
 bounds
     The extents of the resource to select in 2 or 3 dimensions, expressed as a string, e.g.: ``([xmin, xmax], [ymin, ymax], [zmin, zmax])``.  If omitted, the entire dataset will be selected.
 
+spacing
```

Comment:
To me, this hard to figure out for the user.  You have to know how things are arranged internally.  I'm not even sure how your example works (how do you get from 10 to .001 pts. per meter?).  Can you do something like limit the point count for the query area and let the user do the math if they want to, say, get X points/meter?  It's going to be up to the user anyway since they know the SRS, I assume.

Reply:
Initially I had implemented this as a `resolution2d` and `resolution3d` option, but this seemed even more confusing.  To me, spacing is simple: in my example, I want to select increasing resolutions until my points are 10m apart, and then stop.  That corresponds to `1/10^3 = .001` points/m^3 or `1/10^2 = .01` points/m^2 in terms of resolution.

Spacing corresponds pretty much 1:1 to something like `scale` values for LAS, which to me is why it seems simpler than an arg corresponding `resolution`.  A spacing of `1`, with units of meters, is simply a 1m grid, for example - trivial.  My description could be better but I think it's the most intuitive way to present such an option.

Reply:
I still find this confusing.  Specifically, I shouldn't have to know anything about the internal storage of EPT in order to get a reasonable query result.  Mentioning octrees isn't really helpful to most people.  You've been doing octrees for years now, so it's second nature, but I don't think it's something to expect others to know or care about.

I guess I like the resolution notion better.  Call it scale and for all practical purposes it maps to LAS directly, I think.  `.1` gives you a grid of decimeters, `.01` centimeters.

Reply:
Removed mention of the octree, and clarified the meaning of the spacing parameter more clearly, in 5fccc4d040629992792d8a5ad5305e14c1262f25.